src/PageContent/AssignVolunteers/ConfirmationCard.js

```diff
@@ -35,41 +36,45 @@ class ConfirmationCard extends Component {
                     </div>
                     <div className="delivery-details">
                         <h5>Student Deliverers</h5>
-                        <h4>{this.state.studentGroup}</h4>
-                        <h6 className="det">{this.props.deliverer1.name} ({this.props.deliverer1.phone})</h6>
-                        <h6 className="det">{this.props.deliverer2.name} ({this.props.deliverer2.phone})</h6>
+                        <h4>{delivererGroup.group}</h4>
+                        <h6 className="det">{deliverers[0].name} ({deliverers[0].phone})</h6>
+                        <h6 className="det">{deliverers[1].name} ({deliverers[1].phone})</h6>
                     </div>
                 </div>
                 <div className="details-content">
                     <div className="icon-content">
                     </div>
                     <div className="delivery-details">
                         <h5>Dining Hall</h5>
-                        <h4>{this.state.donatingAgency.agency}</h4>
-                        <h6 className="det">{this.state.donatingAgency.address}</h6>
-                        <h6 className="det">{this.state.donatingAgency.primaryContact.name} ({this.state.donatingAgency.primaryContact.phone})</h6>
+                        <h4>{`Name: ${donatingAgency.name}`}</h4>
+                        <h6 className="det">{`Contact: ${daContact.name} (${daContact.phone})`}</h6>
+                        <h6 className="det">{donatingAgency.address.street1}</h6>
+                        <h6 className="det">{`${donatingAgency.address.city}, ${donatingAgency.address.state} ${donatingAgency.address.zipcode}`}</h6>
                     </div>
                 </div>
                 <div className="details-content">
                     <div className="icon-content">
                     </div>
                     <div className="delivery-details">
                         <h5>Recipient</h5>
-                        <h4>{this.state.receivingAgency.agency}</h4>
-                        <h6 className="det">{this.state.receivingAgency.primaryContact.name} ({this.state.receivingAgency.primaryContact.phone})</h6>
+                        <h4>{receivingAgency.agency}</h4>
```

Comment:
just `receivingAgency`

Reply:
I don't know if I should be more impressed that you caught this or more disappointed that I didn't...